src/frontend/Komponenter/Behandling/SettPåVent/LokalkontorOppgavevalg.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { FC, useState } from 'react';
+import { CheckboxGroup, Checkbox, BodyShort } from '@navikt/ds-react';
+import styled from 'styled-components';
+import { dagensDatoFormatert, formaterIsoDato } from '../../../App/utils/formatter';
+
+const FlexBox = styled.div`
+    display: flex;
+    align-items: center;
+    gap: 0.5rem;
+`;
+
+export type Oppgavestatus = {
+    vurderHenvendelseOppgave: VurderHenvendelseOppgavetype;
+    datoOpprettet: string;
+};
+
+type Props = {
+    aktuelleOppgaver: VurderHenvendelseOppgavetype[];
+    oppgavestatus: Oppgavestatus[];
+    settOppgaverMotLokalkontor: (oppgaverMotLokalkontor: VurderHenvendelseOppgavetype[]) => void;
+    oppgaverMotLokalkontor: VurderHenvendelseOppgavetype[];
+    erBehandlingPåVent: boolean;
+};
+
+export const LokalkontorOppgavevalg: FC<Props> = ({
+    aktuelleOppgaver,
+    oppgavestatus,
+    settOppgaverMotLokalkontor,
+    oppgaverMotLokalkontor,
+    erBehandlingPåVent,
+}) => {
+    const tidligereSendteLokalkontorOppgaver = (): VurderHenvendelseOppgavetype[] => {
+        return oppgavestatus.map((oppgave) => {
+            return oppgave.vurderHenvendelseOppgave;
+        });
+    };
+
+    const [checkedOppgaver, settCheckedOppgaver] = useState([
+        ...tidligereSendteLokalkontorOppgaver(),
+        ...oppgaverMotLokalkontor,
+    ]);
+    const erSendt = (oppgave: VurderHenvendelseOppgavetype): boolean => {
+        if (tidligereSendteLokalkontorOppgaver().includes(oppgave)) {
+            return true;
+        }
+        return false;
```

Comment:
`Includes` returnerer allerede `true/false`
```suggestion
    const erSendt = (oppgave: VurderHenvendelseOppgavetype): boolean => tidligereSendteLokalkontorOppgaver().includes(oppgave)
```